### DISCUSSION TITLE
MDN Offline - Native app experience

### DISCUSSION THREAD

**Discussion by brucestull** [mdn/mdn-community#62] [MDN Plus] [answered]
I'm looking at the [MDN Offline page](https://developer.mozilla.org/en-US/plus/docs/features/offline) and there is a phrase:

"You can also add it to your homescreen for an experience similar to using a native app."

What does that mean? I know how to add bookmarks, favorites, etc in browsers. But I'm not sure what this means.

![image](https://user-images.githubusercontent.com/47562501/166624077-8f4d30c2-3f89-4701-a611-0f09bd8d9f8e.png)

**Comment by caugner** [comment, marked as answer]
Thank you for this question, @brucestull. 👋 

> "You can also add it to your homescreen for an experience similar to using a native app."

The phrase refers to the [Add to home screen](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Add_to_home_screen) feature, which is available mainly on mobile devices:

> A2HS is supported in all mobile browsers, except iOS webview. It's also supported in some Chromium desktop browsers.

**Reply by brucestull** [reply]
Thanks , @caugner, for the explanation. Since I'm currently on iOS, I used a shortcut for now. But will use A2HS if I ever switch back to Android.
